dvtemplates/build_k8s_cluster.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Build a K8s cluster from kubevirtci
```

Comment:
Please `set -e` in all shell scrips so default is to exit on error. If you need to allow a command to fail, we should address those individually.

Reply:
Done

Reply:
👍 We're going to want to remove the `x` on these before merge, to prevent exposing any sensitive data.